libs/blocks/merch-card/merch-card.js

```diff
@@ -117,27 +119,63 @@ const getPodType = (styles) => styles?.find((style) => CARD_TYPES.includes(style
 const isHeadingTag = (tagName) => /^H[2-5]$/.test(tagName);
 const isParagraphTag = (tagName) => tagName === 'P';
 
+function transformElement(element) {
+  const div = document.createElement('div');
+  div.innerHTML = element.innerHTML;
+  return div;
+}
+
 const parseContent = (el, merchCard) => {
   const innerElements = [
-    ...el.querySelectorAll('h2, h3, h4, h5, p, ul'),
+    ...el.querySelectorAll('h2, h3, h4, h5, p, ul, em'),
   ];
-  const bodySlot = createTag('div', { slot: 'body-xs' });
+  let bodySlotName = 'body-xs';
+  let headingMCount = 0;
+  if (merchCard.variant === MINI_COMPARE_CHART) {
+    bodySlotName = 'body-m';
+  }
+  const bodySlot = createTag('div', { slot: bodySlotName });
 
   innerElements.forEach((element) => {
     const { tagName } = element;
+    if (element.innerHTML.trim() === '') {
```

Comment:
`element.textContent`? what happens if `element` contains something `<p>&nbsp;</p>` ?

Reply:
Initial thought was to clean the DOM since depending on the authoring there could be empty `<p>` tags. 
Although you bring a good point, removing this to avoid unexpected behavior.